### PR TITLE
RSS reader using JDOM & ROME. Wrt #528 & #557.

### PR DESCRIPTION
**Pull request by jig08**

### CODE REVIEW

File: src/org/loklak/api/search/RSSReader.java [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+		String url = post.get("url", "");
+		
+		URL feedUrl = null;
+		try {
+			feedUrl = new URL(url);
+		} catch (Exception e) {
+			e.printStackTrace();
+		}
+		
+        SyndFeedInput input = new SyndFeedInput();
```
Comment (daminisatya) [inline src/org/loklak/api/search/RSSReader.java]: Please check the indentation.

**Comment by shivenmian** [comment]
@jig08 You have invalid imports, check travis.

File: src/org/loklak/api/search/RSSReader.java [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+		
+		String[][] result = new String[100][7];
+        //result[][0] = Title
+        //result[][1] = Link
+        //result[][2] = URI
+        //result[][3] = Hash Code
+        //result[][4] = PublishedDate
+        //result[][5] = Updated Date
+        //result[][6] = Description
+
```
Comment (sudheesh001) [inline src/org/loklak/api/search/RSSReader.java]: Formatting here is broken. Why is this array commented out ?

**Comment by shivenmian** [comment]
@jig08 Please add jdom and ROME in ```build.gradle``` and ```build.xml``` as well if you haven't already. Classpath doesn't really do anything.

File: src/org/loklak/api/search/RSSReader.java (hunk 1 above)
Comment (jig08) [inline src/org/loklak/api/search/RSSReader.java]: @daminisatya Yes, shall amend it.

File: src/org/loklak/api/search/RSSReader.java (hunk 2 above)
Comment (jig08) [inline src/org/loklak/api/search/RSSReader.java]: Yes, got that @sudheesh001 
These were just the array headings I kept as a reference for myself, I am removing the `String array` anyway .....

**Comment by jig08** [comment]
@shivenmian Yes, I missed out on adding them in ```build.gradle``` and ```build.xml```

**Comment by daminisatya** [comment]
@jig08 Testing the PR can you squash your commits till then.

File: src/org/loklak/api/search/RSSReader.java [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+		totalEntries = i;
+
+		JSONObject rssFeed = new JSONObject();
+		rssFeed.put("Hackernews RSS Feed", jsonArray);
+
+		// print JSON
+		response.setCharacterEncoding("UTF-8");
+		PrintWriter sos = response.getWriter();
+		sos.print(rssFeed.toString(2));
+		sos.println();
```
Comment (daminisatya) [inline src/org/loklak/api/search/RSSReader.java]: @jig08 Since you are trying to form an object and print, did you register your servlet?

Comment (jig08) [inline src/org/loklak/api/search/RSSReader.java]: @daminisatya Could you help me out with that ?

Comment (shivenmian) [inline src/org/loklak/api/search/RSSReader.java]: @jig08 add ```servletHandler.addServlet(RSSReader.class, <your path>);```

Please check how other servlets have been registered, it's on loklakserver.java

Comment (jig08) [inline src/org/loklak/api/search/RSSReader.java]: @shivenmian Thanks. Done here as https://github.com/loklak/loklak_server/pull/624/commits/88dc0fc354ac130383f47477f5fd3207389e2efe
Let me know if it's okay, shall do the squashing then.

cc: @daminisatya

File: src/org/loklak/api/search/RSSReader.java [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+
+			jsonArray.put(i, jsonObject);
+
+			i++;
+		}
+
+		totalEntries = i;
+
+		JSONObject rssFeed = new JSONObject();
+		rssFeed.put("Hackernews RSS Feed", jsonArray);
```
Comment (daminisatya) [inline src/org/loklak/api/search/RSSReader.java]: @jig08 Is this specific to Hackernews? If so please change the API name.

File: src/org/loklak/LoklakServer.java [hunk 5]
```diff
@@ -543,6 +544,7 @@ private static void setServerHandler(File dataFile){
         servletHandler.addServlet(ProxyServlet.class, "/api/proxy.jpg");
         servletHandler.addServlet(ValidateServlet.class, "/api/validate.json");
         servletHandler.addServlet(GenericScraper.class, "/api/genericscraper.json");
+        servletHandler.addServlet(RSSReader.class, "/api/rssreader.json");
```
Comment (daminisatya) [inline src/org/loklak/LoklakServer.java]: @jig08  If this API is specific to Hackernews, then change the name accordingly.

Comment (jig08) [inline src/org/loklak/LoklakServer.java]: @daminisatya I believe it's not. 
Still, let me test with a few other RSS Feeds, Shall let know...

**Comment by sudheesh001** [comment]
The generic RSS seems to read the given RSS feed but this doesn't parse and scrape the hackernews website. Also any other website also says Hacker News RSS Feed
```
http://localhost:9000/api/rssreader.json?url=http://timesofindia.indiatimes.com/rssfeedstopstories.cms
```

**Comment by jig08** [comment]
@sudheesh001 Yes, I started off with Hackernews, hence. Need to make changes here https://github.com/loklak/loklak_server/pull/624#discussion-diff-68294670R111

It does read Hackernews here [https://news.ycombinator.com/](https://news.ycombinator.com/) wrt #528

**Comment by jig08** [comment]
@sudheesh001 Changed the title of the JSON Object returned.

**Comment by daminisatya** [comment]
@jig08 I have been trying that link of hackernews but getting a server error. Can you please re check your code. Since it is a generic rss feeder, make sure you try out some sample URLs and give the output.

**Comment by jig08** [comment]
@daminisatya Ironically, most of the sample runs were made on [https://news.ycombinator.com/](https://news.ycombinator.com/) only :sweat_smile: 
Shall try afresh ...

**Comment by jig08** [comment]
@daminisatya Works fine for me

Here is the output of a sample run: 

```
{
  "RSS Feed": [
    {
      "RSS Feed": "https://news.ycombinator.com/rss",
      "Description": "SyndContentImpl.mode=null\nSyndContentImpl.type=text/html\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11968389\">Comments<\/a>\n",
      "Published-Date": "Fri Jun 24 15:37:04 IST 2016",
      "Updated-Date": "null",
      "Title": "How to Make an RPG: Release",
      "Hash-Code": "780632365",
      "URI": "http://howtomakeanrpg.com/a/how-to-make-an-rpg-release.html",
      "Link": "http://howtomakeanrpg.com/a/how-to-make-an-rpg-release.html"
    },
    {
      "RSS Feed": "https://news.ycombinator.com/rss",
      "Description": "SyndContentImpl.mode=null\nSyndContentImpl.type=text/html\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11966167\">Comments<\/a>\n",
      "Published-Date": "Fri Jun 24 09:18:57 IST 2016",
      "Updated-Date": "null",
      "Title": "UK votes to leave EU",
      "Hash-Code": "-1909144923",
      "URI": "http://www.bbc.co.uk/news/uk-politics-36615028",
      "Link": "http://www.bbc.co.uk/news/uk-politics-36615028"
    },
    {
      "RSS Feed": "https://news.ycombinator.com/rss",
      "Description": "SyndContentImpl.mode=null\nSyndContentImpl.type=text/html\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11968613\">Comments<\/a>\n",
      "Published-Date": "Fri Jun 24 16:13:08 IST 2016",
      "Updated-Date": "null",
      "Title": "Shipping forgettable microservices with Rust",
      "Hash-Code": "1166725682",
      "URI": "https://precompile.com/2016/06/23/shipping-forgettable-microservices-with-rust.html",
      "Link": "https://precompile.com/2016/06/23/shipping-forgettable-microservices-with-rust.html"
    },
    {
      "RSS Feed": "https://news.ycombinator.com/rss",
      "Description": "SyndContentImpl.mode=null\nSyndContentImpl.type=text/html\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11966570\">Comments<\/a>\n",
      "Published-Date": "Fri Jun 24 10:15:37 IST 2016",
      "Updated-Date": "null",
      "Title": "Alan Kay on the misunderstanding of OOP (1998)",
      "Hash-Code": "-194557458",
      "URI": "http://lists.squeakfoundation.org/pipermail/squeak-dev/1998-October/017019.html",
      "Link": "http://lists.squeakfoundation.org/pipermail/squeak-dev/1998-October/017019.html"
    },
    {
      "RSS Feed": "https://news.ycombinator.com/rss",
      "Description": "SyndContentImpl.mode=null\nSyndContentImpl.type=text/html\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11969541\">Comments<\/a>\n",
      "Published-Date": "Fri Jun 24 18:46:17 IST 2016",
      "Updated-Date": "null",
      "Title": "C14 \u2013 Cheap, secure long term cloud storage",
      "Hash-Code": "40148091",
      "URI": "https://www.online.net/en/c14",
      "Link": "https://www.online.net/en/c14"
    },
    {
      "RSS Feed": "https://news.ycombinator.com/rss",
      "Description": "SyndContentImpl.mode=null\nSyndContentImpl.type=text/html\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11964658\">Comments<\/a>\n",
      "Published-Date": "Fri Jun 24 03:39:41 IST 2016",
      "Updated-Date": "null",
      "Title": "AI, Apple and Google",
      "Hash-Code": "-1698697288",
      "URI": "http://ben-evans.com/benedictevans/2016/6/23/ai-apple-and-google",
      "Link": "http://ben-evans.com/benedictevans/2016/6/23/ai-apple-and-google"
    },
    {
      "RSS Feed": "https://news.ycombinator.com/rss",
      "Description": "SyndContentImpl.mode=null\nSyndContentImpl.type=text/html\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11963528\">Comments<\/a>\n",
      "Published-Date": "Fri Jun 24 00:27:00 IST 2016",
      "Updated-Date": "null",
      "Title": "Fire in Microgravity",
      "Hash-Code": "-1107700564",
      "URI": "https://www.americanscientist.org/issues/feature/2016/1/fire-in-microgravity/1",
      "Link": "https://www.americanscientist.org/issues/feature/2016/1/fire-in-microgravity/1"
    },
    {
      "RSS Feed": "https://news.ycombinator.com/rss",
      "Description": "SyndContentImpl.mode=null\nSyndContentImpl.type=text/html\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11967054\">Comments<\/a>\n",
      "Published-Date": "Fri Jun 24 11:37:45 IST 2016",
      "Updated-Date": "null",
      "Title": "The Surprising Relevance of the Baltic Dry Index",
      "Hash-Code": "428114222",
      "URI": "http://www.newyorker.com/business/currency/the-surprising-relevance-of-the-baltic-dry-index?&currentPage=all",
      "Link": "http://www.newyorker.com/business/currency/the-surprising-relevance-of-the-baltic-dry-index?&currentPage=all"
    },
    {
      "RSS Feed": "https://news.ycombinator.com/rss",
      "Description": "SyndContentImpl.mode=null\nSyndContentImpl.type=text/html\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11961345\">Comments<\/a>\n",
      "Published-Date": "Thu Jun 23 20:23:38 IST 2016",
      "Updated-Date": "null",
      "Title": "Demovfuscator",
      "Hash-Code": "-1621198905",
      "URI": "https://github.com/kirschju/demovfuscator",
      "Link": "https://github.com/kirschju/demovfuscator"
    },
    {
      "RSS Feed": "https://news.ycombinator.com/rss",
      "Description": "SyndContentImpl.mode=null\nSyndContentImpl.type=text/html\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11959547\">Comments<\/a>\n",
      "Published-Date": "Thu Jun 23 13:49:47 IST 2016",
      "Updated-Date": "null",
      "Title": "Solar Impulse completes Atlantic crossing with landing in Seville",
      "Hash-Code": "-904464146",
      "URI": "http://www.bbc.com/news/science-environment-36598140",
      "Link": "http://www.bbc.com/news/science-environment-36598140"
    },
    {
      "RSS Feed": "https://news.ycombinator.com/rss",
      "Description": "SyndContentImpl.mode=null\nSyndContentImpl.type=text/html\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11964673\">Comments<\/a>\n",
      "Published-Date": "Fri Jun 24 03:41:16 IST 2016",
      "Updated-Date": "null",
      "Title": "Is productivity the victim of its own success?",
      "Hash-Code": "414698178",
      "URI": "https://growthecon.com/blog/Baumol/",
      "Link": "https://growthecon.com/blog/Baumol/"
    },
    {
      "RSS Feed": "https://news.ycombinator.com/rss",
      "Description": "SyndContentImpl.mode=null\nSyndContentImpl.type=text/html\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11963548\">Comments<\/a>\n",
      "Published-Date": "Fri Jun 24 00:29:52 IST 2016",
      "Updated-Date": "null",
      "Title": "APL compiler targeting a typed array intermediate language",
      "Hash-Code": "509816763",
      "URI": "https://github.com/melsman/apltail/",
      "Link": "https://github.com/melsman/apltail/"
    },
    {
      "RSS Feed": "https://news.ycombinator.com/rss",
      "Description": "SyndContentImpl.mode=null\nSyndContentImpl.type=text/html\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11963409\">Comments<\/a>\n",
      "Published-Date": "Fri Jun 24 00:13:00 IST 2016",
      "Updated-Date": "null",
      "Title": "Open source template for building static web sites with Prismic.io and AWS S3",
      "Hash-Code": "-1461135899",
      "URI": "http://futurice.com/blog/cms-done-right-vol-3",
      "Link": "http://futurice.com/blog/cms-done-right-vol-3"
    },
    {
      "RSS Feed": "https://news.ycombinator.com/rss",
      "Description": "SyndContentImpl.mode=null\nSyndContentImpl.type=text/html\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11969271\">Comments<\/a>\n",
      "Published-Date": "Fri Jun 24 18:06:54 IST 2016",
      "Updated-Date": "null",
      "Title": "Libfiu \u2013 Fault injection in userspace",
      "Hash-Code": "1433997500",
      "URI": "https://blitiri.com.ar/p/libfiu/",
      "Link": "https://blitiri.com.ar/p/libfiu/"
    },
    {
      "RSS Feed": "https://news.ycombinator.com/rss",
      "Description": "SyndContentImpl.mode=null\nSyndContentImpl.type=text/html\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11963369\">Comments<\/a>\n",
      "Published-Date": "Fri Jun 24 00:07:54 IST 2016",
      "Updated-Date": "null",
      "Title": "Show HN: Dynamic drawing application in Swift, inspired by Bret Victor talk",
      "Hash-Code": "-1433004052",
      "URI": "https://github.com/laszlokorte/reform-swift",
      "Link": "https://github.com/laszlokorte/reform-swift"
    },
    {
      "RSS Feed": "https://news.ycombinator.com/rss",
      "Description": "SyndContentImpl.mode=null\nSyndContentImpl.type=text/html\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11968741\">Comments<\/a>\n",
      "Published-Date": "Fri Jun 24 16:34:29 IST 2016",
      "Updated-Date": "null",
      "Title": "The Bitcoin Halving Explained (Infographic)",
      "Hash-Code": "-190177080",
      "URI": "http://coinjournal.net/halvening-explained-infographic/",
      "Link": "http://coinjournal.net/halvening-explained-infographic/"
    },
    {
      "RSS Feed": "https://news.ycombinator.com/rss",
      "Description": "SyndContentImpl.mode=null\nSyndContentImpl.type=text/html\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11969162\">Comments<\/a>\n",
      "Published-Date": "Fri Jun 24 17:51:30 IST 2016",
      "Updated-Date": "null",
      "Title": "TDD in a REPL, Continued: Read-Eval-PrettyPrint-Loop",
      "Hash-Code": "1864185756",
      "URI": "https://spin.atomicobject.com/2016/06/24/read-eval-prettyprint-loop/",
      "Link": "https://spin.atomicobject.com/2016/06/24/read-eval-prettyprint-loop/"
    },
    {
      "RSS Feed": "https://news.ycombinator.com/rss",
      "Description": "SyndContentImpl.mode=null\nSyndContentImpl.type=text/html\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11957471\">Comments<\/a>\n",
      "Published-Date": "Thu Jun 23 03:47:20 IST 2016",
      "Updated-Date": "null",
      "Title": "A new crop of hands-on universities",
      "Hash-Code": "1095362538",
      "URI": "http://www.economist.com/news/international/21701081-latest-push-disrupt-higher-education-new-crop-hands-universities",
      "Link": "http://www.economist.com/news/international/21701081-latest-push-disrupt-higher-education-new-crop-hands-universities"
    },
    {
      "RSS Feed": "https://news.ycombinator.com/rss",
      "Description": "SyndContentImpl.mode=null\nSyndContentImpl.type=text/html\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11963362\">Comments<\/a>\n",
      "Published-Date": "Fri Jun 24 00:06:35 IST 2016",
      "Updated-Date": "null",
      "Title": "Crispr Wins Key Approval to Fight Cancer in Human Trials",
      "Hash-Code": "1396073155",
      "URI": "http://www.bloomberg.com/news/articles/2016-06-23/crispr-wins-key-approval-to-fight-cancer-in-human-trials",
      "Link": "http://www.bloomberg.com/news/articles/2016-06-23/crispr-wins-key-approval-to-fight-cancer-in-human-trials"
    },
    {
      "RSS Feed": "https://news.ycombinator.com/rss",
      "Description": "SyndContentImpl.mode=null\nSyndContentImpl.type=text/html\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11958682\">Comments<\/a>\n",
      "Published-Date": "Thu Jun 23 09:04:11 IST 2016",
      "Updated-Date": "null",
      "Title": "How a comment on Hacker News led to 4½ new Unicode characters",
      "Hash-Code": "711422644",
      "URI": "http://unicodepowersymbol.com/we-did-it-how-a-comment-on-hackernews-lead-to-4-%C2%BD-new-unicode-characters/",
      "Link": "http://unicodepowersymbol.com/we-did-it-how-a-comment-on-hackernews-lead-to-4-%C2%BD-new-unicode-characters/"
    },
    {
      "RSS Feed": "https://news.ycombinator.com/rss",
      "Description": "SyndContentImpl.mode=null\nSyndContentImpl.type=text/html\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11963492\">Comments<\/a>\n",
      "Published-Date": "Fri Jun 24 00:22:52 IST 2016",
      "Updated-Date": "null",
      "Title": "Hydrothermal vent models and the origins of life",
      "Hash-Code": "250146300",
      "URI": "http://nautil.us/issue/37/currents/the-fly-in-the-primordial-soup",
      "Link": "http://nautil.us/issue/37/currents/the-fly-in-the-primordial-soup"
    },
    {
      "RSS Feed": "https://news.ycombinator.com/rss",
      "Description": "SyndContentImpl.mode=null\nSyndContentImpl.type=text/html\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11963268\">Comments<\/a>\n",
      "Published-Date": "Thu Jun 23 23:56:50 IST 2016",
      "Updated-Date": "null",
      "Title": "Containers vs. Config Management",
      "Hash-Code": "-686342950",
      "URI": "http://ciand.cd/28TFehs",
      "Link": "http://ciand.cd/28TFehs"
    },
    {
      "RSS Feed": "https://news.ycombinator.com/rss",
      "Description": "SyndContentImpl.mode=null\nSyndContentImpl.type=text/html\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11969425\">Comments<\/a>\n",
      "Published-Date": "Fri Jun 24 18:26:22 IST 2016",
      "Updated-Date": "null",
      "Title": "Many UK voters didn\u2019t understand Brexit, Google searches suggest",
      "Hash-Code": "-1557835874",
      "URI": "http://arstechnica.com/tech-policy/2016/06/brexit-google-search-trends-tech/",
      "Link": "http://arstechnica.com/tech-policy/2016/06/brexit-google-search-trends-tech/"
    },
    {
      "RSS Feed": "https://news.ycombinator.com/rss",
      "Description": "SyndContentImpl.mode=null\nSyndContentImpl.type=text/html\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11969029\">Comments<\/a>\n",
      "Published-Date": "Fri Jun 24 17:25:29 IST 2016",
      "Updated-Date": "null",
      "Title": "Sanders says he will vote for Clinton",
      "Hash-Code": "-922882653",
      "URI": "http://www.bbc.com/news/world-us-canada-36622383?ocid=socialflow_facebook&ns_mchannel=social&ns_campaign=bbcnews&ns_source=facebook",
      "Link": "http://www.bbc.com/news/world-us-canada-36622383?ocid=socialflow_facebook&ns_mchannel=social&ns_campaign=bbcnews&ns_source=facebook"
    },
    {
      "RSS Feed": "https://news.ycombinator.com/rss",
      "Description": "SyndContentImpl.mode=null\nSyndContentImpl.type=text/html\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11963386\">Comments<\/a>\n",
      "Published-Date": "Fri Jun 24 00:10:06 IST 2016",
      "Updated-Date": "null",
      "Title": "BoiledCarrot",
      "Hash-Code": "1091297870",
      "URI": "http://martinfowler.com/bliki/BoiledCarrot.html",
      "Link": "http://martinfowler.com/bliki/BoiledCarrot.html"
    },
    {
      "RSS Feed": "https://news.ycombinator.com/rss",
      "Description": "SyndContentImpl.mode=null\nSyndContentImpl.type=text/html\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11969070\">Comments<\/a>\n",
      "Published-Date": "Fri Jun 24 17:34:40 IST 2016",
      "Updated-Date": "null",
      "Title": "Games and Culture in Afrika",
      "Hash-Code": "-657864210",
      "URI": "http://weekly.iafrikan.com/9-adebayo-adegbembo-on-games-and-culture-in-afrika/",
      "Link": "http://weekly.iafrikan.com/9-adebayo-adegbembo-on-games-and-culture-in-afrika/"
    },
    {
      "RSS Feed": "https://news.ycombinator.com/rss",
      "Description": "SyndContentImpl.mode=null\nSyndContentImpl.type=text/html\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11968977\">Comments<\/a>\n",
      "Published-Date": "Fri Jun 24 17:17:33 IST 2016",
      "Updated-Date": "null",
      "Title": "Show HN: Decentralized networking library for (not only) games",
      "Hash-Code": "2137764654",
      "URI": "https://github.com/inetic/club",
      "Link": "https://github.com/inetic/club"
    },
    {
      "RSS Feed": "https://news.ycombinator.com/rss",
      "Description": "SyndContentImpl.mode=null\nSyndContentImpl.type=text/html\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11961432\">Comments<\/a>\n",
      "Published-Date": "Thu Jun 23 20:30:32 IST 2016",
      "Updated-Date": "null",
      "Title": "Twilio opens trading at $23.99 per share",
      "Hash-Code": "1863834832",
      "URI": "https://techcrunch.com/2016/06/23/twilio-opens-trading/",
      "Link": "https://techcrunch.com/2016/06/23/twilio-opens-trading/"
    },
    {
      "RSS Feed": "https://news.ycombinator.com/rss",
      "Description": "SyndContentImpl.mode=null\nSyndContentImpl.type=text/html\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11968932\">Comments<\/a>\n",
      "Published-Date": "Fri Jun 24 17:10:17 IST 2016",
      "Updated-Date": "null",
      "Title": "Property-Based Testing with Clojure and Datomic",
      "Hash-Code": "-1454453438",
      "URI": "http://blog.altometrics.com/2016/06/property-based-testing-with-clojure-and-datomic-part-1/",
      "Link": "http://blog.altometrics.com/2016/06/property-based-testing-with-clojure-and-datomic-part-1/"
    },
    {
      "RSS Feed": "https://news.ycombinator.com/rss",
      "Description": "SyndContentImpl.mode=null\nSyndContentImpl.type=text/html\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11968868\">Comments<\/a>\n",
      "Published-Date": "Fri Jun 24 16:59:19 IST 2016",
      "Updated-Date": "null",
      "Title": "Open Source Games You Could Contribute To",
      "Hash-Code": "-791233260",
      "URI": "http://thomasvanderberg.nl/blog/open-source-games-contribute/",
      "Link": "http://thomasvanderberg.nl/blog/open-source-games-contribute/"
    }
  ]
}
```

**Comment by daminisatya** [comment]
@jig08 Can you send me the screenshot on the browser? using the api.

**Comment by jig08** [comment]
@daminisatya Here you go :grinning: 

screenshot from 2016-06-24 19 39 11

**Comment by daminisatya** [comment]
Ok cool. Please squash your commits ASAP and it's good to go.

**Comment by shivenmian** [comment]
@jig08 Why does the ```description``` field have code in it?

Also, com.sun.syndication is an invalid commit, which is being added to the description field.

**Comment by jig08** [comment]
@daminisatya @sudheesh001 Commits squashed. Merging this.

**Comment by shivenmian** [comment]
@jig08 see comment above

**Comment by jig08** [comment]
@shivenmian Sorry, missed that. Have created an issue for the same https://github.com/loklak/loklak_server/issues/633
Shall check it out later, high priority tasks for now.

**Pull request merged**